pkgs/tools/networking/ghz/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ghz";
+  version = "0.109.0";
+
+  src = fetchFromGitHub {
+    owner = "bojand";
+    repo = "ghz";
+    rev = "v${version}";
+    sha256 = "sha256-5l2PeN+VxTaORAkmAfI9TCGd4W6y8BFs/eY4T9nYJuc=";
+  };
+  proxyVendor = true;
```

Are you sure `proxyVendor` is required here?
```suggestion
  proxyVendor = true;
  
```

---

I am not certain, but IIUC `proxyVendor` ensures that the checksum matches even when operating on case-insensitive filesystems. The [go.sum](https://github.com/bojand/ghz/blob/0d77cec5cfe83f13b3f5f877f9ab0a375369ecc9/go.sum) lists some dependencies with uppercase characters in the import path, so I figured I would proxy to be safe.

---

Removed.